recipes/jthread-lite/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(jthread-lite REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} nonstd::jthread-lite)
```

Comment:
```suggestion
target_link_libraries(${PROJECT_NAME} nonstd::jthread-lite)
set_target_properties(${PROJECT_NAME} PROPERTIES CXX_STANDARD 11)
```

Reply:
why C++11 standard if it's C++20 feature?

Reply:
The idea is to provide a jthread class to use with C++11 and later.

Reply:
❤️ Making my 9-5 life much easier. Thanks for another great library!